src/main/java/gov/nasa/cumulus/metadata/aggregator/constant/MENDsIsoXMLSpatialTypeConstant.java

```diff
@@ -0,0 +1,13 @@
+package gov.nasa.cumulus.metadata.aggregator.constant;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class MENDsIsoXMLSpatialTypeConstant {
```

Comment:
It's not clear to me why we need this and `MENDsIsoXMLSpatialTypeEnum`

Reply:
The constant is to define all the possible values within isoXmlSpatial:["footprint", "orbit", "bbox"] jsonArray - which are, footprint, orbit, bbox all as String.  Code logic use the Constants to somehow translate input JsonArray into Hashset<MENDsIsoXMLSpatialTypeEnum> which later will be used to switch on/off footprint/orbit/bbox processing.

Hashset<MENDsIsoXMLSpatialTypeEnum> limited the Hashset to only possible values:
MENDsIsoXMLSpatialTypeEnum.NONE
MENDsIsoXMLSpatialTypeEnum.FOOTPRINT
MENDsIsoXMLSpatialTypeEnum.ORBIT
MENDsIsoXMLSpatialTypeEnum.BBOX

We could  define a HashSet<String> to store the on/off switches about footprint/orbit/bbox without using MENDsIsoXMLSpatialTypeEnum.  However it will leave the room for any developer pushing different String value into HashSet.  For instance Hashset.add("MYString"), which could introduce further confusion to the code.